dev-scripts/openshift_deploy.sh

```diff
@@ -91,27 +91,32 @@ OPENSHIFT_FLAVOR=${OPENSHIFT_FLAVOR:-${DEFAULT_OPENSHIFT_FLAVOR}}
 
 
 if [ "${OPENSHIFT_FLAVOR}" == "minishift" ]; then
-  # ---------------------------
-  # Set minishift configuration
-  # ---------------------------
-  echo -n "[CHE] Checking if minishift is running..."
-  minishift status | grep -q "Running" ||(echo "Minishift is not running. Aborting"; exit 1)
-  echo "done!"  
-
-  DEFAULT_OPENSHIFT_ENDPOINT="https://$(minishift ip):8443/"
+  if [ -z "${MINISHIFT_IP}" ]; then
+    # ---------------------------
```

Comment:
I would prefer if we could avoid nested `if`. And it looks like if env variable `${MINISHIFT_IP}` is set we are not going to verify if minishift is running anymore. 

Why don't you just check if minishift is running and then:
```
MINISHIFT_IP=${MINISHIFT_IP:-$(minishift ip)}
```

Reply:
Yes, if you use it 'remotely', you may not have `minishift` command available. So the status check as it is written won't work.

Reply:
Ok I'm trying to understand here because you usually don't use minishift remotely. You want to use minishift from within a container (e.g. from Che) right? In this case why don't you just use ocp flavor? I'm saying that because it's confusing: if the flavor minishift is select I expect the command minishift to be available. That may be possible if you use the changes proposed [here](https://github.com/redhat-developer/rh-che/pull/340#pullrequestreview-63538831)

Reply:
The reason is that I would like to reuse the exact same configuration ( 
  OPENSHIFT_ENDPOINT, OPENSHIFT_USERNAME, OPENSHIFT_PASSWORD,  OPENSHIFT_NAMESPACE_URL, CHE_KEYCLOAK_DISABLED, etc ...) wheras with `ocp` flavor, I would have to reconfigure all of it manually as far as I understood.